bin/oneoff/data_fix/reset_race_data

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+#
+# Script to reset the race data for the subset of users affected by the race
+# interstitial bug
+#
+
+require_relative '../../../dashboard/config/environment'
+
+File.readlines(ARGV[0]).each do |line|
+  id = line.to_i
+  user = User.find_by_id(id)
+  if user.nil?
+    puts("user #{id} - ERROR - does not exist")
+    next
+  end
+
+  unless user.races
+    puts("user #{id} - ERROR - has no race data")
+    next
+  end
+
+  user.races = nil
+  user.save!
+  puts("user #{id} - success")
+rescue
+  puts("user #{id} - ERROR - other")
```

Comment:
Doubt there will be any errors persisting the record, but if there are it would be handy to out the error message here.

Reply:
My plan is that if there are errors here, to rerun this script over those users with more useful output